contracts/DCAFeeManager/DCAFeeManager.sol

```diff
@@ -12,7 +12,50 @@ contract DCAFeeManager is Governable, IDCAFeeManager {
   /// @inheritdoc IDCAFeeManager
   IDCAHub public immutable hub;
 
-  constructor(IDCAHub _hub, address _governor) Governable(_governor) {
+  TargetTokenShare[] internal _distribution;
+
+  constructor(
+    IDCAHub _hub,
+    TargetTokenShare[] memory _distributionToSet,
```

Comment:
is `calldata` not allowed in constructor?

Reply:
No, must be memory. Not sure why